### DISCUSSION TITLE
Crash between primal tech and bewitchment

### DISCUSSION THREAD

**Discussion by NordicGamerFE** [ACGaming/UniversalTweaks#570] [Bugfix Suggestions]
I've got this weird crash between primal tech and bewitchment
https://mclo.gs/yofEacx
I am to dumb to figure out the exact issue and both mods have been long abadoned on 1.12.2 i am hoping you can find a way to fix this issue
i get this problem when i try to add the carpet to the bewitchment altar close to a low grade charcoal
i am working on Ocean Outlast v.80 but the problem should be present in version v.79A as i have not made any large changes in version 80 so far.

**Comment by NordicGamerFE** [comment]
I just tested and i can confirm the crash happends when only bewitchment legacy, and primal tech is installed

**Comment by ACGaming** [comment]
Looked into it:

The crash occurs because [Bewitchment expects `EnumFacing.Axis` in Primal Tech's `BlockCharcoal`](https://github.com/Um-Mitternacht/Bewitchment/blob/1.12.2/src/main/java/com/bewitchment/common/block/tile/entity/TileEntityWitchesAltar.java#L204), but [Primal Tech is overriding this property with `BlockLog.EnumAxis`](https://github.com/vadis365/PrimalTech/blob/1.12/src/main/java/primal_tech/blocks/BlockCharcoal.java#L22).

As many mods could mess with blockstate properties in unknown ways, fixing Bewitchment's state conversion would be the wisest choice.

**Comment by NordicGamerFE** [comment]
here is an another one for you u https://mclo.gs/l4GGZUF

**Comment by NordicGamerFE** [comment]
Recently got back into working on my pack again, and this is still an issue even tho i seen a fix been pushed that should probably have fixed this
